docs/api.md

```diff
@@ -118,8 +121,46 @@ type AddonsSpec struct {
 }
 
 // TODO maybe we have profiles for scaling behaviors
-type AutoScalerAddonSpec struct {
-    ResourceLimits LimitRange
+type ClusterAutoscaling struct {
+	// Maximum number of nodes in all node groups.
+	// Cluster autoscaler will not grow the cluster beyond this number.
+	// +kubebuilder:validation:Minimum=0
+	MaxNodesTotal int32 `json:"maxNodesTotal"`
+
+	// Should CA scale down the cluster
+    // default: true
+	ScaleDownEnabled bool `json:"enabled,omitempty"`
+
+	// Gives pods graceful termination time before scaling down
+    // default: 600 seconds
```

Comment:
Indentation on all of the default lines seems to be wrong, needs fixing up across this struct